docs/source/aim_sdk.rst

```diff
@@ -1,16 +1,6 @@
 aim.sdk subpackage
```

Comment:
These files are generated by Sphinx when running `sphinx apidoc` command. The naming follows python modules. Maybe we should keep that to not have conflicts when re-generating docs skeleton?

Reply:
hmm, does it generate for the api? 
maybe. we could generate in a separate folder as well.

Reply:
I mean, the whole `source` directory in docs/ is the sphinx stuff. We still do some tweaks (for example, skip some source-code directories for now). there, but I think we should keep the file naming as it is.

Reply:
yea, but in other places I saw (such as PL) the source was the whole docs. I doubt all of it was generated. moreover, in PL they had handpicked classes in the API reference.

Reply:
ok, lets see how it goes in a long run.